scaffolding/templates/action.js.md

```diff
@@ -13,7 +13,9 @@
 
 ### arg2
 
-## Return
+## Resolve
```

Comment:
I'm not entirely convinced this is the right way to document returned values in JS. Then again, I don't know if there is a right way to do it. 
Anyway, I agree with the addition of Resolve & Reject sections, but I've mixed feelings about removing the Return one.

Reply:
This is just the default template because most of time we want to document the resolved or rejected values.  
When a method simply return a value we can manually replace theses sections by the `Return` section